README.md

```diff
@@ -65,25 +65,29 @@ sudo yum install gcc-c++ automake libtool pkgconfig boost-devel \
 To install on a FreeBSD system, use
 
 ```sh
-pkg install devel/git devel/autoconf devel/automake devel/gmake devel/libtool \
+pkg install devel/git devel/cmake
   textproc/expat2 graphics/geos graphics/proj databases/postgresql94-client \
   devel/boost-libs lang/lua52 devel/pkgconf
 ```
 
-Then you should be able to bootstrap the build system:
+CMake 2.8.7 is needed to build oslm2pgsql. First generate the makefiles in separate folder:
 
-    ./autogen.sh
+    mkdir build && cd build
+    cmake .. -G "Unix Makefiles"
```

Comment:
Usually it is needed (many Visual Studios, 32/64 bits are also there, someone prefer nmake makefiles), but this does not affects Travis CI and readme. apvveyor.yml have such option for Visual Studio 64 bit build.

On Windows you have to specify many options dependint on machine beacuse there are no standard pakages and locations for Boost, Expat, etc. I keep them in https://github.com/alex85k/winbuilds , there are other specialized coolections.